backend/app/services/share/knowledge_share_service.py

```
@@ -82,14 +101,35 @@ def _get_resource(
             .first()
         )
         if member:
+            logger.info(
+                f"[_get_resource] User has explicit shared access: member_id={member.id}"
+            )
             return kb
 
+        logger.warning(f"[_get_resource] User has NO explicit shared access")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)